heltour/tournament/models.py

```diff
@@ -242,8 +243,9 @@ def __init__(self, *args, **kwargs):
         self.initial_start_date = self.start_date
         self.initial_is_completed = self.is_completed
 
-    def last_season_alternates(self):
-        last_season = Season.objects.filter(league=self.league, start_date__lt=self.start_date) \
+    def last_season_alternates(self) -> set[Player]:
+        start_date = self.start_date if self.start_date is not None else timezone.now()
```

Comment:
`start_date = self.start_date or timezone.now()`
I slightly prefer this, so long as ruff allows it.